ajax/libs/buy-button-js/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "buy-button-js",
+  "filename": "buybutton.min.js",
+  "description": "BuyButton.js is a highly customizable UI library for adding ecommerce functionality to any website.",
+  "author": "Shopify Inc.",
+  "version": "0.4.13",
```

Comment:
We are almost there! Let's remove the version line as we didn't really add its assets here, we can expect this removal will make the test passed! 😄